cdap-app-fabric/src/main/java/co/cask/cdap/metadata/profile/ProfileMetadataMessageProcessor.java

```diff
@@ -147,7 +147,7 @@ private void updateProfileMetadata(EntityId entityId, MetadataMessage message) {
         }
 
         // Now we only support profile on Workflow type
```

Comment:
The comment needs to be updated

Reply:
Thanks, removed comment since the function comments should be enough__